src/components/Restaurants/RestaurantReview.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { FaSearch } from 'react-icons/fa'; 
+import './RestaurantList.css';
+
+const SearchBar = () => (
+  <header className="header">
+    <div className="searchContainer">
+      <input
+        type="text"
+        placeholder="Restaurants"
+        className="searchInput"
+        aria-label="Search for restaurants"
+      />
+      <input
+        type="text"
+        placeholder="Address, neighborhood"
+        className="locationInput"
+        aria-label="Location"
+      />
+      <button className="searchButton">
+        <FaSearch />
+      </button>
+    </div>
+  </header>
+);
+
+const Filters = () => (
+  <div className="filters">
+    <div className="filterSection">
+      <button className="button">$</button>
+      <button className="button">$$</button>
+      <button className="button">$$$</button>
+      <button className="button">$$$$</button>
+    </div>
+    <div className="filterSection">
+      <label className="checkboxLabel">
+        <input type="checkbox" className="checkbox" />
+        Open Now
+      </label>
+      <label className="checkboxLabel">
+        <input type="checkbox" className="checkbox" />
+        Reservations
+      </label>
+    </div>
+    <div className="filterSection">
+      <button className="categoryButton">Mexican</button>
+      <button className="categoryButton">Sandwiches</button>
+    </div>
+  </div>
+);
+
+const RestaurantEntry = ({ imageSrc, name, tags, rating, reviewCount, url }) => (
+  <div className="restaurantEntry">
```

Comment:
restaurantEntry should be a flex box with some padding so that the image is on the right and the restaurantDetails are on the right
(div: flex, padding: 4px)